edx_event_bus_kafka/internal/producer.py

```diff
@@ -38,6 +38,10 @@
     confluent_kafka = None
 
 
+class EventProductionException(Exception):
```

Comment:
Should these live in a public module?

Reply:
I don't think it needs to, since this is custom to the producing code, and we already had the pattern of keeping consumer exceptions in the `consumer.py` file. I am open to disagreements, though.